components/ws-manager/pkg/manager/monitor.go

```diff
@@ -1016,6 +1020,11 @@ func (m *Monitor) finalizeWorkspaceContent(ctx context.Context, wso *workspaceOb
 					return true, nil, err
 				}
 				readyVolumeSnapshot = true
+				hist, err := m.manager.metrics.volumeSnapshotTimeHistVec.GetMetricWithLabelValues(wsType)
+				if err != nil {
+					log.WithError(err).WithField("type", wsType).Warn("cannot get volume snapshot time histogram metric")
+				}
+				hist.Observe(time.Since(volumeSnapshotTime).Seconds())
```

Comment:
if err is not nil, can we even call Observe? can `hist` be nil? 
maybe should put it into `else` branch?

Reply:
Thank you, I've addressed it.